src/dev/impl/DevToys/Helpers/HashingHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+using DevToys.Models;
+using DevToys.Shared.Core;
+
+namespace DevToys.Helpers
+{
+    internal static class HashingHelper
+    {
+        internal static async Task<byte[]> ComputeHashAsync(
+        HashAlgorithm hashAlgorithm,
+        Stream stream,
+        IProgress<HashingProgress> progress,
+        CancellationToken cancellationToken = default,
```

Comment:
Ideally, a `CancellationToken` should never be an optional argument. Making it optional encourage future maintainer at bad practice by omitting it.

Reply:
Good point, removed the default value.